<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlForm</class>
 <widget class="QWidget" name="ControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>465</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>237</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="grpControls">
       <property name="title">
        <string>Controls</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="grpRepresentation">
          <property name="title">
           <string>Representation</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="rdoPoints">
             <property name="text">
              <string>Points</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdoWireframe">
             <property name="text">
              <string>Wireframe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdoSolid">
             <property name="text">
              <string>Solid</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpOpacity">
          <property name="title">
           <string>Opacity</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSlider" name="sliderOpacity">
             <property name="value">
              <number>99</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpRotation">
          <property name="title">
           <string>Rotation</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QRadioButton" name="rdoManual">
             <property name="text">
              <string>Manual</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rdoAutomatic">
             <property name="text">
              <string>Automatic</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QFrame" name="grpAxes">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="chkX">
                <property name="text">
                 <string>X</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkY">
                <property name="text">
                 <string>Y</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chkZ">
                <property name="text">
                 <string>Z</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rdoWireframe</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onWireframe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoSolid</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onSolid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoPoints</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onPoints()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderOpacity</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ControlForm</receiver>
   <slot>onOpacityChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoManual</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onManualRotation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoAutomatic</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onAutomaticRotation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkX</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onAutoRotX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkY</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onAutoRotY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkZ</sender>
   <signal>clicked()</signal>
   <receiver>ControlForm</receiver>
   <slot>onAutoRotZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onWireframe()</slot>
  <slot>onSolid()</slot>
  <slot>onPoints()</slot>
  <slot>onOpacityChange(int)</slot>
  <slot>onManualRotation()</slot>
  <slot>onAutomaticRotation()</slot>
  <slot>onAutoRotX()</slot>
  <slot>onAutoRotY()</slot>
  <slot>onAutoRotZ()</slot>
 </slots>
</ui>
